h2o-r/h2o-DESCRIPTION.template

```diff
@@ -26,6 +26,7 @@ Imports: graphics,
 Suggests: ggplot2,
           mlbench,
           Matrix,
+          slam,
```

Comment:
Yes, I think I will add that for both slam and data.table to check them.  If one of the package is not there, I will use the h2o default as.h2o instead.